diff -uNr a/msrest/authentication.py b/msrest/authentication.py
--- a/msrest/authentication.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/authentication.py	2019-05-03 15:24:56.197378655 +0200
@@ -23,6 +23,8 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Optional, Dict
 
 import requests
@@ -250,4 +252,4 @@
                 self._topic_key_header: topic_key,
             }
         )
-    
\ No newline at end of file
+    
diff -uNr a/msrest/configuration.py b/msrest/configuration.py
--- a/msrest/configuration.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/configuration.py	2019-05-03 15:21:21.714180695 +0200
@@ -33,6 +33,8 @@
     from ConfigParser import NoOptionError  # type: ignore
 import platform
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Dict, List, Any, Callable
 
 import requests
diff -uNr a/msrest/exceptions.py b/msrest/exceptions.py
--- a/msrest/exceptions.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/exceptions.py	2019-05-03 15:24:04.788289959 +0200
@@ -27,6 +27,7 @@
 import logging
 import sys
 
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Callable, Any, Optional, TYPE_CHECKING
 
 if TYPE_CHECKING:
diff -uNr a/msrest/http_logger.py b/msrest/http_logger.py
--- a/msrest/http_logger.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/http_logger.py	2019-05-03 15:25:23.071902273 +0200
@@ -28,6 +28,8 @@
 import re
 import types
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Any, Union, Optional, TYPE_CHECKING
 
 if TYPE_CHECKING:
diff -uNr a/msrest/paging.py b/msrest/paging.py
--- a/msrest/paging.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/paging.py	2019-05-03 15:21:59.087518189 +0200
@@ -29,6 +29,8 @@
 except ImportError:
     from collections import Iterator
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Dict, Any, List, Callable, Optional, TYPE_CHECKING
 
 if TYPE_CHECKING:
diff -uNr a/msrest/pipeline.py b/msrest/pipeline.py
--- a/msrest/pipeline.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/pipeline.py	2019-05-03 15:23:44.789644462 +0200
@@ -33,6 +33,8 @@
     from urllib.parse import urlparse
 import xml.etree.ElementTree as ET
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Dict, Any, Optional, Union, List, TYPE_CHECKING
 
 if TYPE_CHECKING:
diff -uNr a/msrest/polling/poller.py b/msrest/polling/poller.py
--- a/msrest/polling/poller.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/polling/poller.py	2019-05-03 15:22:48.745637924 +0200
@@ -31,6 +31,8 @@
 except ImportError:
     from urllib.parse import urlparse
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Any, Callable, Union, List, Optional, TYPE_CHECKING
 
 if TYPE_CHECKING:
diff -uNr a/msrest/serialization.py b/msrest/serialization.py
--- a/msrest/serialization.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/serialization.py	2019-05-03 15:24:30.140840555 +0200
@@ -41,6 +41,7 @@
 
 import isodate
 
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Dict, Any
 
 from .exceptions import (
diff -uNr a/msrest/service_client.py b/msrest/service_client.py
--- a/msrest/service_client.py	2018-07-13 00:15:27.000000000 +0200
+++ b/msrest/service_client.py	2019-05-03 15:23:16.304149413 +0200
@@ -33,6 +33,8 @@
     from urllib.parse import urljoin, urlparse
 import warnings
 
+import sys
+sys.path.insert(0, '/usr/lib/python-msrest/bundled')
 from typing import Any, Dict, Union, IO, Tuple, Optional, cast, TYPE_CHECKING
 
 if TYPE_CHECKING:
